web/template/home.gohtml

```diff
@@ -290,8 +292,266 @@
+                                                                               :class="vod.Dropdown.getChild('downloads').value ? 'fa-chevron-up' : 'fa-chevron-down'"></i>
+                                                                        </button>
+                                                                        <div x-cloak
+                                                                             x-show="vod.Dropdown.getChild('downloads').value"
+                                                                             class="grid gap-1">
+                                                                            <template x-for="file in vod.Downloads">
+                                                                                <a href="" download
+                                                                                   class="block text-left hover:bg-gray-100 dark:hover:bg-gray-800 py-2 px-4"
+                                                                                   x-text="file.FriendlyName">
+                                                                                </a>
+                                                                            </template>
+                                                                        </div>
+                                                                    </div>
+                                                                </template>
+                                                            {{end}}
+                                                        </div>
+                                                    </template>
+                                                </div>
+                                            </article>
+                                        </template>
+                                    </article>
+                                    <template x-if="courseStreams.hasNext()">
+                                        <div class="flex flex-grow justify-center mt-8">
+                                            <button class="tum-live-button tum-live-button-secondary" type="button"
+                                                    @click="courseStreams.next()">
+                                                <i class="fa-solid fa-angles-down mr-2"></i>
+                                                More
+                                            </button>
+                                        </div>
+                                    </template>
+                                </section>
+                            </section>
+                        </template>
+                    </div>
+                    <template x-if="plannedStreams.hasElements()">
+                        <section class="tum-live-course-view-item xl:col-span-1">
+                            <header>
+                                <h3>Upcoming</h3>
```

Comment:
In the test database I added a lecture that starts in 5 minutes for the "Einführung in Brauereiwesen" course. However, I only see the course after that in upcoming.

Reply:
That's because lecture that are going live in the next 30 minutes aren't shown as planned anymore. 

Code:
```golang
func (s Stream) IsPlanned() bool {
        //                                                  ⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄⌄
	return !s.Recording && !s.LiveNow && !s.IsPast() && !s.IsComingUp()
}
```

This problem will vanish once waiting rooms are properly implemented.

Reply:
Okay, I see!